webapp/src/sidepanel.tsx

```diff
@@ -187,28 +171,27 @@ export class Sidepanel extends data.Component<SidepanelProps, SidepanelState> {
                         {showFileList && <filelist.FileList parent={parent} />}
                         {showFullscreenButton && <div id="miniSimOverlay" role="button" title={lf("Open in fullscreen")} onClick={this.handleSimOverlayClick} />}
                     </div>
-                    {isTabTutorial && <div className="tutorial-controls">
-                        {backButton}
-                        <Button icon="lightbulb" disabled={true} className="tutorial-hint" />
-                        {nextButton}
-                    </div>}
-                </TabContent>
-                {tutorialOptions && <TabContent name={TUTORIAL_TAB} icon="icon tasks" showBadge={activeTab !== TUTORIAL_TAB} onSelected={this.tryShowTutorialTab} ariaLabel={lf("Open the tutorial tab")}>
-                    <TutorialContainer
-                        parent={parent}
-                        tutorialId={tutorialOptions.tutorial}
-                        name={tutorialOptions.tutorialName}
-                        steps={tutorialOptions.tutorialStepInfo}
-                        currentStep={tutorialOptions.tutorialStep}
-                        tutorialOptions={tutorialOptions}
-                        hideIteration={tutorialOptions.metadata?.hideIteration}
-                        hasTemplate={!!tutorialOptions.templateCode}
-                        preferredEditor={tutorialOptions.metadata?.preferredEditor}
-                        onTutorialStepChange={onTutorialStepChange}
-                        onTutorialComplete={onTutorialComplete}
-                        setParentHeight={this.setComponentHeight} />
-                </TabContent>}
-            </TabPane>
+                </div>
+            </div>
+            {tutorialOptions &&
+                <div id="tutorialWrapper" className={this.props.tutorialSimSidebar ? "topInstructionsWrapper" : ""} style={tutorialHeightStyle}>
+                    <div className={outerTutorialContainerClassName} style={tutorialHeightStyle}>
```

Comment:
Would the style not be inherited from the parent div above? I'm wondering why we need to define the style again here.

Reply:
Weirdly, it isn't inherited, but while looking into it I realized it can be simplified a little by just putting the style on the inner div and moving the id there too, so I'll make that change.